lua/bufferline/render.lua

```diff
@@ -975,18 +728,15 @@ function render.update(update_names, refocus)
   )
 
   if not ok then
-    render.disable()
+    command('BarbarDisable')
```

Comment:
Can't run `events.disable()` because `events` imports `render`, so if `render` imports `events` that'd be a dependency cycle. Since the `:BarbarDisable` command decouples the two it seemed like this is a fine workaround